Microsoft.Toolkit.Uwp.UI.Controls.Layout/ContentSizer/ContentSizer.Events.cs

```diff
@@ -88,15 +88,18 @@ private bool VerticalMove(double verticalChange)
                 return true;
             }
 
+            verticalChange = InvertDragDirection ? -verticalChange : verticalChange;
+
             if (!IsValidHeight(TargetControl, verticalChange))
             {
                 return true;
             }
 
-            // TODO: This only works if splitter is on top and making things grow down
             // Do we need our ContentResizeDirection to be 4 way? Maybe 'Auto' would check the horizontal/vertical alignment of the target???
             TargetControl.Height += verticalChange;
 
+            GripperCursor = Windows.UI.Core.CoreCursorType.SizeNorthSouth;
```

Comment:
Isn't this set when we initialize the properties? I don't think we need to re-set it here as it's just bound in XAML, so this is effectively a no-op, right?